jockeyJSAndroid/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,10 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I've integrated the JockeyJSAndroid project into our project (by migrating the 'src/java' library).
Now it all compiles into the same `.aar` file.